persist/fs/read.go

```diff
@@ -260,7 +271,7 @@ func (r *reader) Read() (ts.ID, []byte, error) {
 
 	r.entriesRead++
 
-	return ts.BinaryID(entry.Id), data, nil
+	return ts.StringID(string(entry.Id)), data, nil
```

Comment:
These string casts here and elsewhere are exactly why binary IDs were introduced in the first place – reverting them back would skyrocket p99s back to multiple seconds.